src/modules/addTodo.js

```diff
@@ -0,0 +1,58 @@
+import todo from './todo';
+
+const input = document.getElementById('add-todo');
+const todosUL = document.getElementById('todos');
+export function removeTodo(e) {
+  if (e.target.classList.contains('remove')) {
+    const todo = e.target.parentElement;
+    todo.parentElement.removeChild(todo);
+  }
```

Comment:
- [ ] This statement is never right, because there is no condition that adds `remove` class, please consider restating the function.

Reply:
I'm done with the proper clear all btn.